app/assets/javascripts/components/jquery_wrappers/AtWhoInput.jsx

```diff
@@ -1,37 +1,32 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 
-class AtWhoInput extends React.Component {
-  constructor(props) {
-    super(props);
-    this.saveTextareaRef = textarea => {
-      this.textarea = textarea;
-    };
-  }
+const AtWhoInput = ({ usernames, name, value, onChange }) => {
+  const textareaRef = useRef(null);
 
-  componentDidMount() {
-    this.loadAtWho();
-  }
+  useEffect(() => {
+    loadAtWho();
+  }, [value]);
 
-  loadAtWho() {
-    $(this.textarea)
-      .atwho({
-        at: '@',
-        data: this.props.usernames,
-      })
-      .on('inserted.atwho', this.props.onChange);
-  }
+  const loadAtWho = () => {
+    if (textareaRef.current) {
+      $(textareaRef.current)
+        .atwho({
+          at: '@',
+          data: usernames,
+        })
+        .on('inserted.atwho', onChange);
+    }
+  };
```

Comment:
What do you think about looking for a replacement library to use instead of using a jQuery adapter for that?